src/styles/index.scss

```diff
@@ -1,3 +1,61 @@
-body {
+@import 'vars';
+
+.page__body {
   margin: 0;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: row wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: 10px;
+
+  &__day {
+    width: $day-width;
+    height: $day-width;
+    background-color: $calendar-color;
+    color: black;
+    border: $day-border;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+
+    &:hover {
+      cursor: pointer;
+      background-color: $pointer-calendar-color;
+      transform: translateY(-20px);
+      transition-duration: 500ms;
+    }
+
+    @for $numbers from 1 through 31 {
+      &:nth-child(#{$numbers})::before {
+        content: '#{$numbers}';
+      }
+    }
+  }
```

Comment:
The `.calendar__day` class has a lot of styles. Consider breaking it down into smaller, reusable classes or mixins to keep your code DRY (Don't Repeat Yourself). For example, you can create a mixin for flex-centering.